.github/workflows/dependabot.yml

```diff
@@ -0,0 +1,29 @@
+name: Dependabot auto-merge
+
+on:
+  pull_request:
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  dependabot:
+    runs-on: ubuntu-latest
+    if: ${{ github.actor == 'dependabot[bot]' }}
+    steps:
+      - id: metadata
+        uses: dependabot/fetch-metadata@v1
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Enable auto-merge for Dependabot PRs
+        if: >-
+          ${{
+            contains(steps.metadata.outputs.dependency-names, 'my-dependency')
```

Comment:
Is `'my-dependency'` some magic value?

Reply:
I *think* that’s just a copy-pasta error. Will fix it.